src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/HistoricalQuestion.kt

```diff
@@ -123,4 +125,10 @@ class HistoricalQuestion(
   override fun toString(): String {
     return "HistoricalQuestion(code='$code', responses=$responses)"
   }
+
+  fun toDto() = HistoricalQuestionDto(
+    code = code,
+    question = question,
+    responses = responses.map { it.toDto() },
```

Comment:
I like the `responses = responses.map(HistoricalResponse::toDto),` form better because it makes it much clearer what are the types you're dealing with.